new/LIT7089QeneSena.xml

```diff
@@ -0,0 +1,88 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7089QeneSena" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስነ፡ ጽጌረዳ፡ 
+                    ዓለም፡ እንተ፡ ትትነገፊ፡ ወተሐልፊ፡ ቅጽበተ፡ አርአያ፡ ጽላሎት፡ ወህልም፤</title>
```

Comment:
Let us use in the title ፡ at the end of the transcription of t1, it is better. Also, the first line should not necessarily be quoted in full, sometimes it is better to shorten it (to avoid a break in the middle of a sense unit).